policy.md

```diff
@@ -1,178 +1,116 @@
-  TLS/SSL server certificates)
+* Baseline Requirements (BR) audit (if the certificate is capable of issuing TLS/SSL server certificates)
 * Extended Validation (EV) SSL and/or Code Signing audit (if applicable)
 
 Exceptions to providing audit information:
-
-* The SHA-256 fingerprint of the certificate is specifically listed as in scope 
-in the audit statements of the parent certificate, and the 
-"Audits Same as Parent" checkbox is checked; or
+* The SHA-256 fingerprint of the certificate is specifically listed as in scope in the audit statements of the parent certificate, and the “Audits Same as Parent” checkbox is checked; or
 * The certificate has expired; or
-* The certificate is technically-constrained as described in section 7.1.5 
-of the CA/Browser Forum Baseline Requirements, or
-* The certificate has been revoked, and the corresponding record in the CCADB 
-has been updated with the correct revocation status.
-
-CAs must provide English versions of any Certificate Policy, Certification
-Practice Statement and Audit documents which are not originally in English,
-with version numbers matching the document they are a translation of. The
-English version is not required to be authoritative in all cases of dispute,
-but the CA must attest that the translation is not materially different to the
-original.
+* The certificate is technically-constrained as described in section 7.1.5 of the CA/Browser Forum Baseline Requirements, or
+* The certificate has been revoked, and the corresponding record in the CCADB has been updated with the correct revocation status.
+
+CAs must provide English versions of any CP, CPS and Audit documents which are not originally in English, with version numbers matching the document they are a translation of. The English version is not required to be authoritative in all cases of dispute, but the CA must attest that the translation is not materially different to the original.
+
+CAs must submit an [Add/Update Root Request Case](https://docs.google.com/document/d/1ttmeeqO6WxDWe_deDNsGUgDO_LpsvoduFNZeHHMw_f8/edit) to add or update audit information stored in the CCADB.
 
 ### 5.1 Audit Statement Content ###
 
-CCADB uses an Audit Letter Validation (ALV) tool to automatically parse and validate audit statements. This system eliminates manual processing, but it requires audit statements to follow some basic rules in order to function properly. If the audit statement fails to meet any of the following requirements, the CA will be asked to work with their auditor to provide an audit statement that passes ALV.
+CCADB uses an Audit Letter Validation (ALV) tool to automatically parse and validate audit statements. This system eliminates manual processing, but it requires audit statements to follow some basic rules in order to function properly. If the audit statement fails to meet any of the following requirements, the CA must work with their auditor to provide an audit statement that passes ALV.
```

Comment:
Maybe we strike "eliminates manual processing, but it"?  I don't think the statement is entirely accurate, and it isn't necessary to convey the rest of the intent.